configs/production.js.save

```diff
@@ -47,9 +47,8 @@ config.server.host = config.server.domain;
 config.mongodb.name = 'veres_one';
 config.mongodb.host = 'localhost';
 config.mongodb.port = 27017;
-config.mongodb.local.collection = 'veres_one';
-config.mongodb.username = 'veres';
-config.mongodb.password = null;
+//config.mongodb.username = 'veres';
+//config.mongodb.password = null;
```

Comment:
I don't know what's going on here, can you leave a comment or fix it?

Reply:
added comments on mongodb auth options here: https://github.com/veres-one/veres-one/pull/44/commits/551293071163a115939c7319c3ad5f22b69292d0